### DISCUSSION TITLE
Way to make "weekly" habits?

### DISCUSSION THREAD

**Discussion by FireDragons52** [iSoron/uhabits#1587] [Help & FAQ]
I want habits that ask questions but only on Saturdays. I've set it up for only Saturdays but I realize now that that's for the reminders, it still shows the habit every single day.

Is there a way to make the check and question only on the Saturdays too?

**Comment by iSoron** [comment]
From the FAQ:

> ### How can I specify that a habit should be performed on certain days of the week?
>
> You don't need to specify this. The app will figure it out by itself. You just need to specify how many times would you like to repeat the habit in a certain interval of time. For example, if you want to repeat a certain habit every Monday and Friday, simply specify that you want to repeat the habit 2 times per week, and then start practicing your habit. If, during your first week, you indeed perform your habit on Monday and Friday, the app will automatically add a checkmark on the following Saturday and Sunday. As long as you keep your routine, the checkmarks should appear automatically on the correct days.
